static/src/js/components/GranuleImage/GranuleImage.js

```diff
@@ -13,128 +13,99 @@ import './GranuleImage.scss'
  * @param {Object} props - The props passed into the component.
  * @param {String} imageSrc - The image source.
```

Comment:
This should be updated, it is no longer how we comment nested parameters. See ABExperiement.js for an example

Reply:
changed